service/rpc/client.go

```diff
@@ -15,6 +15,7 @@ type RPCClient struct {
 	addr       string
 	processPid int
 	client     *rpc.Client
+	evalScope  api.EvalScope
```

Comment:
To be specific, this is the piece of state I'd like to see if we can get rid of. I _really_ _really_ like the EvalScope abstraction and I like the way the rest of the code is developed such that they are more generalized.

The issue to me is, we're keeping this state here on `c.evalScope`, and is this state that other clients will have to manage as well? If so, let's move it somewhere higher (i.e. the Debugger struct). If this is state that is only going to be needed by this client, essentially only for the terminal, then fine.

Does this make sense?